duplicacy/duplicacy_main.go

```diff
@@ -1403,6 +1421,11 @@ func main() {
 					Usage:    "the RSA public key to encrypt file chunks",
 					Argument: "<public key>",
 				},
+				cli.StringFlag{
+					Name:     "erasure-coding",
+					Usage:    "enable erasure coding to protect against storage corruption",
```

Comment:
About naming: 

From https://forum.duplicacy.com/t/new-feature-erasure-coding/4168/2?u=thebestpessimist i understand that this only protects the same chunk from corruption, and not the whole storage (ie. like raid5/6 does).

I think the Usage should be reworded to something like `enable erasure coding to protect against intra-chunk corruption`, as this is not "storage level protection", just "chunk-level protection".


What would happen if in the future you would also add something like storage-level protection via something similar to raid5/6, which would do parity for any chunk?